sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/unnest/BeamZetaSqlUnnestRule.java

```diff
@@ -58,10 +58,6 @@ public void onMatch(RelOptRuleCall call) {
     RelNode outer = call.rel(1);
     RelNode uncollect = call.rel(2);
 
-    if (correlate.getCorrelationId().getId() != 0) {
```

Comment:
This is fixed just by fixing the convert calls? Cool!

Reply:
Thanks for the review. Yeah that's the only bug that blocked this feature.